bin/akula-rpc.rs

```diff
@@ -43,15 +124,572 @@ where
             .unwrap_or(BlockNumber(0)))
     }
 
+    async fn call(&self, call_data: CallData, block_number: BlockNumber) -> RpcResult<Bytes> {
+        let block_hash = canonical_hash::read(&self.db.begin().await?, block_number)
+            .await?
+            .unwrap();
+
+        let header = header::read(&self.db.begin().await?, block_hash, block_number)
+            .await?
+            .unwrap();
+
+        let tx = &self.db.begin().await?;
+        let mut state = Buffer::new(tx, BlockNumber(0), Some(block_number));
+        let mut analysis_cache = AnalysisCache::default();
+        let block_spec = MAINNET.collect_block_spec(block_number);
+
+        let value = match call_data.value {
```

Comment:
Here and everywhere: `unwrap_or(some_value)` or `unwrap_or_default()`

Reply:
Even for types like Box<dyn Consensus> used as engine in creating an ExecutionProcessor?